lib/spirv/test/BUILD.gn

```diff
@@ -15,12 +15,4 @@ if (enable_unittests) {
       "//third_party/swiftshader_flutter:spvtools_val",
     ]
   }
-
-  executable("glsl_to_spirv") {
```

Comment:
Why can't this entire GN file be removed?

Reply:
The [program above](https://github.com/flutter/engine/blob/d0ec2205b34e89f4f8c887d1d378abad91ed9df9/lib/spirv/test/spirv_assembler.cc) is still used. It looks like it is used for assembling invalid bytecode to make sure that the transpiler throws errors correctly.